src/patternfly/base/tokens/tokens-default.scss

```diff
@@ -438,6 +454,8 @@
   --pf-t--global--spacer--action--horizontal--plain: var(--pf-t--global--spacer--sm);
   --pf-t--global--spacer--action--horizontal--spacious: var(--pf-t--global--spacer--xl);
   --pf-t--global--spacer--action--vertical--compact: var(--pf-t--global--spacer--xs);
+  --pf-t--global--spacer--action--vertical--default: var(--pf-t--global--spacer--sm);
+  --pf-t--global--spacer--action--vertical--plain: var(--pf-t--global--spacer--sm);
```

Comment:
@lboehling can you confirm these are intentional? Last I remember we were using `--pf-t--global--spacer--control--vertical[...]` for the top/bottom padding of actions/controls so that we used a single spacer.